src/neo/Network/P2P/Payloads/FilterAddPayload.cs

```diff
@@ -1,10 +1,17 @@
+using Neo.Cryptography;
```

Comment:
mistake?
```suggestion
```

Reply:
It's not a mistake. The XML comment needs it.